x/twasm/contract/callback_msgs.go

```diff
@@ -31,8 +31,10 @@ type BeginBlock struct {
 
 type EvidenceType string
 
-const EvidenceDuplicateVote EvidenceType = "DuplicateVote"
-const EvidenceLightClientAttack EvidenceType = "LightClientAttack"
+const (
```

Comment:
Ah, these were defined improperly in Go, but work in Rust?

Of course, the [auto-snake-casing](https://github.com/confio/poe-contracts/blob/main/packages/bindings/src/sudo.rs#L47-L65)

Glad you tested this and good fix.